Automattic/wp-calypso
# Jetpack Cloud: minify missing-credentials.svg + style improvements

#### Changes proposed in this Pull Request

Looks like this file is executable, it is commented here: https://github.com/Automattic/wp-calypso/pull/41561#issuecomment-633566902

- Created a new svg file and minified it
- Applied some styles because the size of the image was interfering in the styles.

Before:
![image](https://user-images.githubusercontent.com/9832440/82927951-5ab6a580-9f79-11ea-8985-e1bcf4592fc4.png)

After:
![image](https://user-images.githubusercontent.com/9832440/82927984-6bffb200-9f79-11ea-80e2-4d822702036b.png)


#### Testing instructions

- Apply the changes 
- Use a site without your server credentials
- Check if the icon is the correct and it's not executable
- Check the styles, icon centered aligned with the title.
